gen/modules/general.h

```diff
@@ -279,3 +281,30 @@ struct user_desc {
 #define BLKSSZGET  0x1268
 #define BLKPBSZGET 0x127B
 #endif
+
+struct msghdr {
+    void         *msg_name;
+    socklen_t     msg_namelen;
+    struct iovec *msg_iov; 
+    size_t        msg_iovlen;
+    void         *msg_control;
+    size_t        msg_controllen;
+    int           msg_flags;
```

Comment:
In the kernel, `msg_flags` is `unsigned` rather than `int`, `msg_namelen` is `int` rather than `socklen_t`.

Reply:
fixed